openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -317,75 +317,62 @@ def fill_company_account_cash_basis_base_account_id(env):
     )
 
 
-def fix_group_company_and_create_groups_for_each_company_if_necessary(env):
-    def _get_all_ordered_children(groups):
-        children = groups
-        for child_group in groups:
-            group_childs = env["account.group"].search(
-                [("parent_id", "in", child_group.ids)]
-            )
-            if group_childs:
-                children |= _get_all_ordered_children(group_childs)
-        return children
-
-    all_parent_groups = env["account.group"].search([("parent_id", "=", False)])
-    compute_parent_path = False
-    for parent_group in all_parent_groups:
-        group_plus_childs = _get_all_ordered_children(parent_group)
-        accounts = env["account.account"].search(
-            [("group_id", "in", group_plus_childs.ids)]
-        )
-        companies = accounts.mapped("company_id")
-        if len(companies) == 1:
-            group_plus_childs.filtered(lambda g: g.company_id != companies).write(
-                {"company_id": companies.id}
+def populate_account_groups(env):
+    """Generate the generic account groups for each company. Later code will
+    do it for manually created groups.
+    """
+    companies = env["res.company"].with_context(active_test=False).search([])
```

Comment:
@pedrobaeza There is no `active` field on the `res.company`, so context `active_test` is not needed

Reply:
It doesn't hurt. You can have this module installed (if migrated to 14.0): https://odoo-community.org/shop/company-active-4952#attr=6890

Even more, such context is propagated downstream, and can serve for other searches inside the scope. And it's a good practice in OpenUpgrade to always pass this context for avoiding to let data unprocessed.

Anyways, that's not the main focus of this PR. What is your problem with this?

Reply:
Thanks for your information, I have no problem